doc/src/sgml/installation-meson.sgml

```diff
@@ -1104,6 +1104,48 @@ meson configure ... --with-llvm LLVM_CONFIG='/path/to/llvm/bin/llvm-config'
 
      <variablelist>
 
+      <varlistentry>
+       <term><option>--auto_features=<replaceable>enabled/disabled/auto</replaceable></option></term>
```

Comment:
I wonder if we should change the enabled/disabled/auto bit in various places to list auto/ first, given it's the default?

Reply:
It is in most cases, so makes sense to change the order. However, just wanted to note that it's not the default in all. eg. llvm, dtrace.